routes/board.js

```diff
@@ -152,6 +153,21 @@ Router.route('/boards/:board_id')
 				return next(utils.error(400, err));
 			}
 			Board.populate(board, 'createdBy', function(err, board) {
+
+				if(req.resolved.board.size.width < old.size.width || req.resolved.board.size.height < old.size.height){
+					Ticket.find({ 'board': req.resolved.board.id,
+						$or: [
+								{'position.x': {$gt: (req.resolved.board.size.width * 192) - 96}},
```

Comment:
What are these magic numbers? Maybe declare them as variables as to make the code more readable.